### ISSUE TITLE
policy_gradient 原理介绍部分内容格式存在问题

### ISSUE THREAD

**Issue by leanna62** [PaddlePaddle/models#797]
https://github.com/PaddlePaddle/models/tree/develop/fluid/policy_gradient 
policy_gradient  demo介绍部分，看起来格式存在问题，能辛苦调整下吗？或者以什么样的方式可以看到原始的文档呢？ @wanghaoshuang  @lcy-seso

**Comment by wanghaoshuang** [comment]
非常抱歉，公式渲染没处理好。
可以先参考这个内容完全一致的版本：https://wanghaoshuang.github.io/2017/12/Policy-Gradient-RL-by-PaddlePaddle/

**Comment by leanna62** [comment]
嗯嗯，收到，非常感谢~

**Comment by leanna62** [comment]
另外，我实验的时候发现 fluid 新版本的代码中，brain.py中47行 loss = fluid.layers.reduce_mean(x=neg_log_prob_weight)  
需要修正为 loss = fluid.layers.reduce_mean(neg_log_prob_weight)  才可以正常运行。

**Comment by wanghaoshuang** [comment]
非常感谢反馈，已提交PR修改。
